<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: /bg_color/;
}</string>
   </property>
   <layout class="QVBoxLayout" name="centralwidget_layout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="navbar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
	border: 0px;
}</string>
      </property>
      <layout class="QVBoxLayout" name="navbar_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="tab_bar" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
	background-color: /tab_bar_bg_color/;
}
QTabBar {
	background: /tab_bar_bg_color/;
	border: 0px;	
	color: white;
}
QTabBar::tab {
    background: /tab_bg_color/;
    border: 2px solid /tab_bar_accent_color/;
    border-radius: 6px;
    padding: 4px;
	margin: 2px;
	color: /tab_font_color/;
}
QTabBar::tab:hover {
	background-color: /tab_hover_color/;
}
QTabBar::tab:selected {
	background: /tab_focus_color/;
}</string>
         </property>
         <layout class="QHBoxLayout" name="tab_bar_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QTabBar" name="tabs" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="window_management_buttons" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QWidget {
	border: 0px;
}</string>
            </property>
            <layout class="QHBoxLayout" name="window_management_buttons_layout">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="mazimize_button">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>png/maximize button icon.png</normaloff>png/maximize button icon.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>23</width>
                 <height>23</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="minimize_button">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>png/minimize button icon.png</normaloff>png/minimize button icon.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>23</width>
                 <height>23</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="close_button">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>png/close button icon.png</normaloff>png/close button icon.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>23</width>
                 <height>23</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="main_bar" native="true">
         <property name="styleSheet">
          <string notr="true">QWidget {
	background-color: /main_bar_bg_color/;
}
QWidget {
	border: 0px;
}
QPushButton {
	background-color: rgba(21, 21, 21, 0);
	color: /main_bar_accent_color/;
	border: 0px;
	font: 24px &quot;dripicons-v2&quot;;
	margin: 2px;
	padding: 2px;
}
QPushButton:hover {
	background-color: /main_bar_hover_color/;
	border-radius: 10px;
}
QPushButton:pressed {
	padding-top: 4px;
	padding-left: 4px;
}
QLineEdit {
	background-color: /url_bar_bg_color/;
	border-radius: 13px;
	color: rgb(255, 255, 255);
	padding-left: 8px;
	padding-bottom: 2px;
	margin-top: 3px;
	margin-bottom: 3px;
	margin-left: 1px;
	margin-right: 1px;
	border: 2px solid /main_bar_accent_color/;
}
QLineEdit:hover {
	background-color: /main_bar_hover_color/;
}
QLineEdit:focus {
	background-color: /main_bar_focus_color/;
}</string>
         </property>
         <layout class="QHBoxLayout" name="main_bar_layout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="back_button">
            <property name="text">
             <string>l</string>
            </property>
            <property name="iconSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="forward_button">
            <property name="text">
             <string>m</string>
            </property>
            <property name="iconSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="reload_and_stop_button">
            <property name="text">
             <string>Z</string>
            </property>
            <property name="iconSize">
             <size>
              <width>23</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="url_bar">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="dragEnabled">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Search or type a URL</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="new_tab_button">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>î€µ</string>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="main_layout">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="bookmarks_sidebar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header>qtabbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
